src/App.js

```diff
@@ -1,17 +1,43 @@
-//App.js
-import React, { Component } from "react";
-import DateInput from "./components/DateInput";
-import Photo from "./components/Photo.js";
-import Explanation from "./components/Explanation";
-
-export default function App() {
-
-    return (
-      <div className="App">
-        <h1>NASA's Astronomy Picture of the Day</h1>
-        <Data />
-        <Photo />
-      </div>
-    );
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import Card from "../src/Card";
+import "./App.css";
 
+
+
+function App() {
+  const [data, setData] = useState({});
+  useEffect(() => {
+    axios
+      .get("https://api.nasa.gov/planetary/apod?api_key=IpSWIoboMnzGCH5JDIFqiGhn16zuwo4kIaBJnHgi&date=2019-07-15")
+      .then(res => {
+        // data receipt successful
+        console.log(res.data);
+        console.log(setData);
+        // receive data
+        setData(res.data);
+
+
+      })
+      .catch(error => {
+        // data receipt failure
+        console.log(error, "Error receiving data");
+      });
+  }, []);
+
+
+
+  return (
```

Comment:
careful with the spacing, that's a lot there. 
side note i do like the commit descriptions